static/generated/privacy.pug

```diff
@@ -65,17 +65,17 @@ html(lang="en")
     p
```

Comment:
This will force a update in the privacy terms for everyone. Not sure if we care that much?

Reply:
It would be best to do the change in a separate PR regardless, that way the user will only see this change and won't have to read through the rest of this PR.

Reply:
Agreed